### DISCUSSION TITLE
How to get started with open source...

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#3217] [Tips and Tricks]
### Steps
1. pick a project / technology - I recommend looking at your friends or community projects or ones you use already
2. look at issues and discussions
3. either pick an existing issue or raise some issues for ideas / suggestions / questions / improvements
4. after discussing in the discussions, move to issues, then fork, make changes, raise PR

### Resources
- Open Source book (a bit out of date, but still useful) https://github.com/eddiejaoude/book-open-source-tips
- Add your name to the list in the README file to practice the workflow http://github.com/EddieHubCommunity/hacktoberfest-practice
- Create a json file for your LinkFree profile (open source LinkTree alternative) https://github.com/EddieHubCommunity/LinkFree

Then start making improvements to the projects, remember it is not just about code, check the documentation, automated tests, can you help on the issues

**Comment by EmmaDawsonDev** [comment]
I see a lot of people coming to the discord and asking about projects in specific languages. I know there's always documentation and things people can do but I think it would be great if we could curate a list of open source projects that regularly have beginner-friendly issues for different languages so we have somewhere to point people looking for things like python or java etc.

**Reply by eddiejaoude** [reply]
Great idea 👍  I think this could be done in a couple ways

1. use threads in this discussion, each for a different tech
2. markdown file on the `support` repo
... other ideas welcome

**Reply by PetraAG** [reply]
Hello everyone,i followed the link from discord. Is the conversation still open? Good morning

**Reply by eddiejaoude** [reply]
I think this conversation is always open, so we can continually improve it
